common/common.scss

```diff
@@ -61,3 +61,17 @@
     }
   }
 }
+
+.alert-peek-mode-dependency {
+  body:not(.staff) &,
```

Comment:
Is it possible to move this logic to JS? Could you check there for "currentUser = staff & body does not have the full-width class"?